### PR TITLE
Misc docs update

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by Xiaoy312 — APPROVED** [review]

**Review by MartinZikmund — APPROVED** [review]

File: doc/articles/get-started-next-steps.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 Already [completed the tutorial](getting-started-tutorial-1.md)? Ready to build your own app or library, or just want to learn more? These resources will help you on your way.
 
-* The Uno Platform brings Microsoft's UWP/WinUI framework to Android, iOS, macOS, Linux, and WebAssembly. The wealth of [official Microsoft documentation on the UWP framework](https://docs.microsoft.com/en-us/windows/uwp/develop/) is a great place to start, particularly if you've never used UWP/WinUI or XAML before. Some additional resources:
+* The Uno Platform brings Microsoft's WinUI framework to Android, iOS, macOS, mac Catalyst, Linux, and WebAssembly. The wealth of [official Microsoft documentation on the WinUI framework](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/) is a great place to start, particularly if you've never used WinUI or XAML before. Some additional resources:
```
Comment (sasakrsmanovic) [inline doc/articles/get-started-next-steps.md:5]: is a great place to start, particularly if you've never used WinUI or XAML before. All concepts, docs and samples translate to Uno Platform.  Some additional resources:

File: doc/articles/how-uno-works.md [hunk 2]
```diff
@@ -51,7 +50,7 @@ The codebase of an Uno Platform application is a mix of XAML markup, C# code, im
 
 ### Binaries
 
-The C# code is the easy part - [.NET runs pretty much everywhere](https://docs.microsoft.com/en-us/dotnet/core/introduction). On iOS, Android, and macOS, Uno.UI is using [Xamarin Native](https://dotnet.microsoft.com/learn/xamarin/what-is-xamarin) (note: not Xamarin.Forms). On the web, it's using .NET running in [WebAssembly](https://webassembly.org/), and on Linux it's running under .NET Core. 
+The C# code is the easy part - [.NET runs pretty much everywhere](https://docs.microsoft.com/en-us/dotnet/core/introduction). On iOS, Android, and macOS, Uno.UI is using [Xamarin Native](https://dotnet.microsoft.com/learn/xamarin/what-is-xamarin) (note: not Xamarin.Forms or MAUI). On the web, it's using .NET running in [WebAssembly](https://webassembly.org/), and on Linux it's running under .NET 6 or later.
```
Comment (sasakrsmanovic) [inline doc/articles/how-uno-works.md:53]: Should we say that Xamarin Native is net6 is is called .net Android and .net iOS

File: doc/articles/how-uno-works.md [hunk 3]
```diff
@@ -4,34 +4,33 @@ How does Uno Platform make the same application code run on all platforms?
...
- panels, control templates, measurement and arranging, etc - is the same. However, it's sometimes useful to know what's going on at render-time, for example if you're [intermixing native views with XAML views](native-views.md) on a platform-specific basis.
+As an application developer, you normally don't need to worry about exactly how Uno.UI renders your visual tree to the screen on each platform. For every platform, the public API - panels, control templates, measurement and arranging, etc - is the same. However, it's sometimes useful to know what's going on at render-time, for example if you're [intermixing native views with XAML views](native-views.md) on a platform-specific basis.
 
 #### Web (WebAssembly)
 
 On the web, each XAML element is converted into an appropriate [HTML element](https://developer.mozilla.org/en-US/docs/Glossary/Element). Panels, controls, and other 'intermediate' elements in the visual tree are converted to `<div/>` elements, whereas 'leaf' elements like `TextBlock`, `Image` etc get converted into more specific tags (`<p/>`, `<img/>` etc).
 
-#### iOS
+#### iOS and mac Catalyst
```
Comment (kazo0) [inline doc/articles/how-uno-works.md:25]: Mac Catalyst?
mac Catalyst?

They both feel weird

Comment (jeromelaban) [reply doc/articles/how-uno-works.md:25]: Based on Apple's docs, it's `Mac Catalyst`, so we can change it everywhere :) I'll take note. Thanks!

**Pull request merged**